pipeline/transformer/transformer_sierra/src/test/scala/weco/pipeline/transformer/sierra/transformers/SierraGenresTest.scala

```diff
@@ -122,44 +143,70 @@ class SierraGenresTest
       )
     )
 
-    SierraGenres(bibData) shouldBe expectedGenres
+    val List(genre) = SierraGenres(bibData)
+
+    genre should have(
+      'label ("A Content - X Content - V Content")
+    )
+
+    val List(conceptA, conceptX, conceptV) = genre.concepts
+    conceptA shouldBe a[Concept[_]]
+    conceptA should have(
+      'label ("A Content"),
+      labelDerivedConceptId("A Content")
+    )
+    conceptX shouldBe a[Concept[_]]
+    conceptX should have(
+      'label ("X Content"),
+      labelDerivedConceptId("X Content")
+    )
+    conceptV shouldBe a[Concept[_]]
+    conceptV should have(
+      'label ("V Content"),
+      labelDerivedConceptId("V Content")
+    )
   }
 
   it("returns subjects for tag 655 with subfields a, y") {
-    val expectedGenres =
-      List(
-        Genre(
-          label = "A Content - Y Content",
-          concepts = List(
-            Concept(label = "A Content"),
-            ParsedPeriod(label = "Y Content")
-          )))
 
     val bibData = createSierraBibDataWith(
       varFields = List(
         VarField(
           marcTag = "655",
           subfields = List(
-            Subfield(tag = "y", content = "Y Content"),
+            Subfield(tag = "y", content = "MDCCLXXXVII. [1787]"),
             Subfield(tag = "a", content = "A Content")
           )
         )
       )
     )
 
-    SierraGenres(bibData) shouldBe expectedGenres
+    val List(genre) = SierraGenres(bibData)
+
+    genre should have(
+      'label ("A Content - MDCCLXXXVII. [1787]")
+    )
+
+    val List(conceptA, conceptV) = genre.concepts
+    conceptA shouldBe a[Concept[_]]
+    conceptA should have(
+      'label ("A Content"),
+      labelDerivedConceptId("A Content")
+    )
+    conceptV shouldBe a[Period[_]]
+    conceptV should have(
+      'label ("MDCCLXXXVII. [1787]"),
+      labelDerivedPeriodId("MDCCLXXXVII. [1787]"),
```

Comment:
I'm not convinced that Periods should have ids, and even if they do, I don't think deriving them from the unmodified label is correct.